src/System.Text.Primitives/Properties/AssemblyInfo.cs

```diff
@@ -10,6 +10,13 @@
 [assembly: AssemblyProduct("System.Text.Primitives")]
 [assembly: AssemblyTrademark("")]
 
+[assembly: InternalsVisibleTo("System.Text.Primitives.Performance.Tests,PublicKey="+
+"0024000004800000940000000602000000240000525341310004000001000100039ac461fa5c82"+
+"c7dd2557400c4fd4e9dcdf7ac47e3d572548c04cd4673e004916610f4ea5cbf86f2b1ca1cb824f"+
+"2a7b3976afecfcf4eb72d9a899aa6786effa10c30399e6580ed848231fec48374e41b3acf88119"+
+"31343fc2f73acf72dae745adbcb7063cc4b50550618383202875223fc75401351cd89c44bf9b50"+
+"e7fa3796")]
+
```

Comment:
This sounds broken to me. I think it is important to expose the highest performance static workhorse methods as public so that folks who want to just encode piece of UTF8 without the overhead of virtual calls, casts and other crap can do so.